pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -55,12 +55,12 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		if name == lastPlanStatus.Name {
 			planDisplay := fmt.Sprintf("Plan %s (%s strategy) [%s]%s", name, plan.Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message))
 			planBranchName := rootBranchName.AddBranch(planDisplay)
-			for _, phase := range lastPlanStatus.Phases {
-				phaseDisplay := fmt.Sprintf("Phase %s [%s]%s", phase.Name, phase.Status, printMessageIfAvailable(phase.Message))
+			for i, lastPhase := range lastPlanStatus.Phases {
+				phaseDisplay := fmt.Sprintf("Phase %s (%s strategy) [%s]%s", lastPhase.Name, plan.Phases[i].Strategy, lastPhase.Status, printMessageIfAvailable(lastPhase.Message))
```

Comment:
although this might work, I would be much calmer if instead of plan.Phases[i] we selected the phase by name (so in a separate function). but I could be persuaded that this is fine :)